TheJumpCloud/support
# Sa 3118 get jc system app

## Issues
* [SA-3118](https://jumpcloud.atlassian.net/browse/sa-3118) - SA-3138_Get-JCSystemApp

## What does this solve?
Merge Get-JCSystemApp to master and PSGallery

## Is there anything particularly tricky?
n/a
## How should this be tested?
Verify Get-JCSystemApp parameters are working properly

[SA-3118]: https://jumpcloud.atlassian.net/browse/SA-3118?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
